indexers/keyvalue/MongoDBIndexer/mongodbhandler.py

```diff
@@ -83,3 +83,19 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.client.close()
         except pymongo.errors.PyMongoError as exp:
             raise MongoDBException(exp)
+
+    def delete(self, keys: Iterator[int], *args, **kwargs):
+        import pymongo
+        try:
+            count = 0
+            for k in keys:
+                result = self.collection.delete_one({'_id': k})
```

Comment:
is there a batch delete?

If we count the number of deletes in this way is because expect it to be different than the amount of deletes we expect right? Then the relevant info to debug is the difference or the ones that you did not manage to delete (or the keys that u did in fact delete)

Reply:
Yep, found the `delete_many`. Needed a special operator `$in`